python/tk_multi_publish2/dialog.py

```diff
@@ -117,7 +117,7 @@ def __init__(self, parent=None):
         self.ui.drag_progress_message.hide()
 
         # buttons
-        self.ui.validate.clicked.connect(self.do_validate)
+        self.ui.validate.clicked.connect(lambda: self.do_validate())
```

Comment:
I am missing a "why" here - if i come back to this code 12 months from now, i will probably think "why is it using a lambda for `do_validate` but not for `do_publish`..

Perhaps can you just add a comment and include some of the explanation and links from your excellent PR description? That will help future readers of the code to understand they why!

Thank you!

Reply:
Thanks for the review. I've added the description in the code.